addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/DataGenerator.java

```diff
@@ -117,12 +123,15 @@ private String generateArrayValue(String name, Parameter parameter) {
     }
 
     public String generateBodyValue(String name, Schema<?> property) {
-        if (isArray(property.getType())) {
+        if (isArray(property)) {
             return generators
                     .getArrayGenerator()
                     .generate(name, (ArraySchema) property, "csv", false);
+        } else if (isMap(property)) {
```

Comment:
The `else` is unnecessary, the above returns (same for the following else).

Reply:
Hi @thc202 , I didn't understand what you are pointing there as I'm seeing three different flows. Could you clarify, please?

Reply:
This is a matter of style, I'm suggesting the following:
```Java
if (isArray(property)) {
    return generators
            .getArrayGenerator()
            .generate(name, (ArraySchema) property, "csv", false);
}
if (isMap(property)) {
    return generators.getMapGenerator().generate(TYPES, property);
}
return generateValue(name, property, false);
```
since the `if`s return the `else`s are unnecessary.

Reply:
ok